### DISCUSSION TITLE
Can I edit wording in Standard Portal

### DISCUSSION THREAD

**Discussion by ryzam** [Budibase/budibase#9579] [Help] [answered]
Hi guys,

Any way without change the source code, I can edit the wording the the Standard Portal

![image](https://user-images.githubusercontent.com/597759/217452919-5977cc38-afd0-424c-88e2-80b41885c00d.png)

**Comment by andz-bb** [comment, marked as answer]
hey @ryzam 

unfortunately this text is currently hardcoded into the portal so there's no way to configure it without modifying the source code. Is this something you would be interested in having a configuration option for in the portal settings?

The [source code for that screen is here](https://github.com/Budibase/budibase/blob/develop/packages/builder/src/pages/builder/apps/index.svelte#L131) if you're interested in changing it yourself 👍
